### PR TITLE
updated and complete french translation

### PR DESCRIPTION
**Pull request by fat115**
Translation is complete and there's no other language than french ;)

Sorry to use a PR instead of Weblate but I have no account ont it and Poedit is really faster and more ergonomic than Weblate.
Hope it will be useful.

### CODE REVIEW

File: locale/fr.po [hunk 1]
```diff
@@ -20,132 +20,123 @@ msgstr ""
 
 #: init.lua
 msgid "Air (you hacker you!)"
-msgstr ""
+msgstr "Air (espèce de tricheur ! )"
```
Comment (lnjX) [inline locale/fr.po:23]: Please remove the spaces before and after the `!`

File: locale/fr.po [hunk 2]
```diff
@@ -954,405 +900,355 @@ msgstr "Pissenlit jaune"
...
 
 #: init.lua
 msgid "Waterlily"
-msgstr ""
+msgstr "Nénuphar"
 
 #: init.lua
 msgid "Ignore (you hacker you!)"
-msgstr ""
+msgstr "Ignorer (espèce de tricheur ! )"
```
Comment (lnjX) [inline locale/fr.po:931]: Here again, please remove the spaces.

**Comment by lnjX** [comment]
PRs in general are no problem. The (free) hosted Weblate is a bit slow and overloaded. But for the account: You can also just login with your GitHub account.
Please correct the mentioned issues, then I can merge it. Thank you for your contribution :)

File: locale/fr.po (hunk 1 above)
Comment (fat115) [reply locale/fr.po:23]: Sorry but it's a french typographic rule : some punctuation signs (colon, exclamation mark, question mark, semicolon, some guillemets) need to be preceded and followed by a space.

File: locale/fr.po (hunk 2 above)
Comment (fat115) [reply locale/fr.po:931]: Same as above ;)

File: locale/fr.po (hunk 1 above)
Comment (lnjX) [reply locale/fr.po:23]: Then it's ok of course! :smiley_cat: :+1:

**Pull request merged**